game.js

```diff
@@ -380,11 +387,22 @@ function main() {
   });
 
   const game = new Game({
-    width: 25,
-    height: 25,
-    squareSize: 20,
+    width: WIDTH,
+    height: HEIGHT,
+    squareSize: SQUARE_SIZE,
     players: [player1, player2],
   });
 
-  game.start();
+  const table = document.getElementById("table-stats");
+  table.width = WIDTH*SQUARE_SIZE;
+  table.height = HEIGHT*SQUARE_SIZE;
+
+  const ui = new UI({
+    game: game,
+    playButton: document.getElementById("play-button"),
+    stopButton: document.getElementById("stop-button"),
+    stepButton: document.getElementById("step-button"),
+    generationCounter: document.getElementById("generation-counter"),
+    events: Events,
```

Comment:
why are you passing a module level enum as a configuration parameter to ur ui? u should just import that